test/test_muscles.cpp

```diff
@@ -26,19 +26,21 @@ static unsigned int muscleForIdealizedActuator(1);
 
 
 TEST(IdealizedActuator, unitTest){
+    biorbd::Model model(modelPathForMuscleForce);
+    biorbd::muscles::IdealizedActuator idealizedActuator(model.
+        muscleGroup(muscleGroupForIdealizedActuator).
+        muscle(muscleForIdealizedActuator));
+    biorbd::rigidbody::GeneralizedCoordinates Q(model);
+    biorbd::rigidbody::GeneralizedVelocity qDot(model);
+    Q = Q.setOnes() / 10;
+    qDot = qDot.setOnes() / 10;
+
```

Comment:
I understand why this can be interesting to factor this out. However, I am worried that it introduces inter-dependencies between different section of the code. Because now, two `{ }` sections share the same muscle that was maybe changed. This isn't the case for this particular one, because you don't change the muscle, but when you test the attributes of the muscle by changing them, these changes will follow... 
Do you have any opinion on that?

Reply:
I wanted to factor it out to avoid all the repetitions of these lines and make the code simpler and lighter. I will let you decide if you want me to keep it or change it back. 
I do understand what you mean by modifying the muscle though and that it could be a potential source of error.

Reply:
@pariterre , do you want me to change it back to the way it was before?

Reply:
Yep.. let's make things not factorized. 
The tests do not need to be light, but accurate